client/src/pages/EventPage/EventDetail.scss

```diff
@@ -6,6 +6,12 @@
   margin-bottom: 40px;
   min-height: 800px;
 
+  &__event-img {
```

Comment:
Useful fix))

Reply:
this is a fix for a map. Your styles for header image redefined styles for icons of map in footer.